src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Build/OutputGroup.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Build
+{
+    /// <summary>
+    /// A set of outputs generated by a project that fit under some common category.
+    /// </summary>
+    [DebuggerDisplay("Output Group {Name} ({TargetName})")]
+    internal class OutputGroup : IOutputGroup
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OutputGroup"/> class.
+        /// </summary>
+        internal OutputGroup(string name, string targetName, string displayName, string? description, IImmutableList<KeyValuePair<string, IImmutableDictionary<string, string>>> items, bool successful)
+        {
+            Requires.NotNullOrEmpty(name, nameof(name));
+            Requires.NotNullOrEmpty(targetName, nameof(targetName));
+            Requires.NotNullOrEmpty(displayName, nameof(displayName));
+            Requires.NotNull(items, nameof(items));
+
+            Name = name;
+            TargetName = targetName;
+            DisplayName = displayName;
+            Description = description;
+            Outputs = items;
+            IsSuccessful = successful;
+        }
+
+        /// <inheritdoc/>
```

Comment:
Remove these <inheritdoc/> they are not needed, VS will automatically show docs from the inteface for them.